etcdctl/command/role_commands.go

```diff
@@ -101,7 +101,7 @@ func actionRoleList(c *cli.Context) {
 	}
 
 	for _, role := range roles {
-		fmt.Printf("%s\n", role)
+		fmt.Printf("%s\n", role.Role)
```

Comment:
Without it, etcdctl will print the dump of the whole structure what looks rather messy.
I think etcdctl should have additional command, eg. role show <role_name> to display details of the selected role.

I can prepare such code.